scripts/docs/installation.js

```diff
@@ -15,49 +15,22 @@ $(function() {
     }
 
     // Setup sliders
-    var verifyDownloadSlider = new Slider({
-        slidesContainer: 'verifying-your-download-slide-container',
-        choicesContainer: 'verifying-your-download-choices-container',
-        id: 'verifying-your-download-choices',
-        choices: ['verifying-your-download-on-windows', 'verifying-your-download-on-os-x', 'verifying-your-download-on-linux'],
-        hideHeadings: true
-    });
-    var createUsbSlider = new Slider({
-        slidesContainer: 'creating-a-usb-choices-slide-container',
-        choicesContainer: 'creating-a-usb-choices-container',
-        id: 'creating-a-usb-choices',
-        choices: ['creating-a-usb-on-windows', 'creating-a-usb-on-others'],
-        hideHeadings: true
-    });
-    var bootSlider = new Slider({
-        slidesContainer: 'booting-choices-slide-container',
-        choicesContainer: 'booting-choices-container',
-        id: 'booting-choices',
-        choices: ['booting-on-a-pc', 'booting-on-a-mac'],
-        hideHeadings: true
+    var operatingSystemSlider = new Slider({
+        slidesContainer: 'installation-instructions-slide-container',
+        choicesContainer: 'operating-system-choices-container',
+        id: 'operating-system-choices',
+        choices: ['install-on-windows', 'install-on-os-x', 'install-on-ubuntu'],
+        hideHeadings: false
     });
 
     // Show instructions for the current platform
     var currentOs = detectOS();
 
     if (currentOs == 'windows' || !currentOs) {
-        verifyDownloadSlider.slideTo('verifying-your-download-on-windows');
+        operatingSystemSlider.slideTo('install-on-windows');
     } else if (currentOs == 'osx') {
-        verifyDownloadSlider.slideTo('verifying-your-download-on-os-x');
-    } else {
-        verifyDownloadSlider.slideTo('verifying-your-download-on-linux');
-    }
-
-    if (currentOs == 'windows' || !currentOs) {
-        createUsbSlider.slideTo('creating-a-usb-on-windows');
+        operatingSystemSlider.slideTo('install-on-os-x');
     } else {
-        createUsbSlider.slideTo('creating-a-usb-on-others');
+        operatingSystemSlider.slideTo('install-on-ubuntu');
     }
-
-    if (currentOs == 'osx') {
-        bootSlider.slideTo('booting-on-a-mac');
-    } else {
-        bootSlider.slideTo('booting-on-a-pc');
-    }
-
 })();
```

Comment:
You are right, the call was there already before I committed my changes and seems not to break anything. However since it is redundant I'll quickly commit a fix with the parentheses removed anyways.